package.json

```diff
@@ -14,5 +19,9 @@
   "homepage": "https://github.com/openedx/brand-openedx#readme",
   "publishConfig": {
     "access": "public"
+  },
+  "devDependencies": {
+    "@openedx/paragon": "^23.0.0"
   }
 }
+ 
```

Comment:
nit: looks like line 27 might be non-empty (e.g., spaces/indent), so GitHub is showing that file needs a blank new line at the end of file.